### ISSUE TITLE
Model not training with CUFED Dataset

### ISSUE THREAD

**Issue by IshikaB3003** [bia006/HiTSR#2]
Hi,
When I tried training the model with the CUFED dataset, the model did not show any significant learning, even after reducing the learning rate by a factor of 10 (from 1e-4 to 1e-5). Is there a recommended setup or any specific configuration needed for training?

**Comment by bia006** [comment]
Due to the model's complexity, training can be slow using a Swin Transformer. To optimize performance, you can remove the interpolated image added in the final layer. Additionally, tweaking the depth of the attention heads may help improve accuracy if resources allow. These tweaks help balance efficiency and model performance. You can try a learning scheduler that dynamically reduces the learning rate based on validation loss too.

**Comment by NghiaZun** [comment]
> Hi, When I tried training the model with the CUFED dataset, the model did not show any significant learning, even after reducing the learning rate by a factor of 10 (from 1e-4 to 1e-5). Is there a recommended setup or any specific configuration needed for training?

Do you have CUFED dataset?
